scripts/analytics/legacy_wikis.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from tabulate import tabulate
+
+from framework.mongo import database
+from website import settings
+from website.addons.wiki import settings as wiki_settings
+
+from .utils import mkdirp
+
+wiki_collection = database['nodewikipage']
+node_collection = database['node']
+
+TAB_PATH = os.path.join(settings.ANALYTICS_PATH, 'tables', 'features')
+mkdirp(TAB_PATH)
+
+def get_wiki_counts():
+
+    query_legacy = {'date': {'$lt': wiki_settings.WIKI_CHANGE_DATE}}
+    wikis_legacy = wiki_collection.find(query_legacy)
+
+    totals = {'pub_proj': {'total': 0, 'total_size': 0, 'largest': 0},
+              'pub_reg': {'total': 0, 'total_size': 0, 'largest': 0},
+              'priv_proj': {'total': 0, 'total_size': 0, 'largest': 0},
+              'priv_reg': {'total': 0, 'total_size': 0, 'largest': 0}}
+
+    metadata = {}
+    for entry in wikis_legacy:
+        node = entry.get('node')
+        page_name = entry.get('page_name')
+        if not node in metadata.keys():
+            node_meta = node_collection.find_one({'_id': node})
+            if node_meta['is_public']:
+                if node_meta['is_registration']:
+                    node_type = 'pub_reg'
+                else:
+                    node_type = 'pub_proj'
+            elif node_meta['is_registration']:
+                node_type = 'priv_reg'
+            else:
+                node_type = 'priv_proj'
+            metadata.update({node: {'current':
+                                          node_meta['wiki_pages_current'],
+                                          'node_type': node_type}})
+
+        if not entry.get('_id') == metadata[node]['current'][page_name]:
```

Comment:
A `KeyError` is being raised here (on `page_name`) when I run this on a prod dump. Is this expected? How should it be handled?